README.md

```diff
@@ -90,32 +90,33 @@ One of the primary advantage of CMake based build system is integration with oth
   pushd build
   ```
 
-3. Run cmake with the appropriate options:
+3. Run cmake with the appropriate options, e.g.:
 
   ```
-  cmake \
+  cmake .. \
    -DNRN_ENABLE_PYTHON=ON \
    -DNRN_ENABLE_CORENEURON=OFF \
    -DNRN_ENABLE_MPI=OFF \
    -DNRN_ENABLE_INTERVIEWS=OFF \
-   ..
   ```
 
 4. Build the code:
 
   ```
   make -j
+  make install
   ```
 
 Particularly useful CMake options are (use **ON** to enable and **OFF** to disable feature):
 
-* **-DNRN\_ENABLE\_INTERVIEWS=ON** : Build Interviews to enable GUI support
+* **-DNRN\_ENABLE\_INTERVIEWS=OFF** : Disable Interviews (native GUI support)
 * **-DNRN\_ENABLE\_PYTHON=ON** : Enable Python support
```

Comment:
this is the default. Might want to reverse this and the others which are mere restatements of the defaults.